ant-networking/src/driver.rs

```diff
@@ -1005,6 +1025,58 @@ impl SwarmDriver {
                         relay_manager.try_connecting_to_relay(&mut self.swarm, &self.bad_nodes)
                     }
                 },
+                Some(()) = Self::conditional_interval(&mut bootstrap_cache_save_interval) => {
+                    let Some(bootstrap_cache) = self.bootstrap_cache.as_mut() else {
+                        continue;
+                    };
+                    let Some(current_interval) = bootstrap_cache_save_interval.as_mut() else {
+                        continue;
+                    };
+                    let start = Instant::now();
+
+                    let config = bootstrap_cache.config().clone();
+                    let mut old_cache = bootstrap_cache.clone();
+
+                    let new = match BootstrapCacheStore::new(config) {
```

Comment:
doesn't need to have a new one ?
just cotinue with the existing one shall be ok?

anyway, this depends on how the BootstrapCache handles `merge`.

Reply:
The existing one will contain the tracked peers and will only be cleared after calling `flush`. That is spawned as a separate task, so we cannot clone it as well. So need a new one here.